Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -555,7 +647,34 @@ def delete_row_with_old_chrome_configurations_from_chrome_instances_file(chrome_
     if index_to_delete >= 0:
         del splitted_chrome_instances_contents[index_to_delete]
         chrome_instances_contents = '\n'.join(splitted_chrome_instances_contents)
-        write_file(CHROME_INSTANCES_FILE_PATH, chrome_instances_contents, overwrite=True)
+        write_text_file(CHROME_INSTANCES_FILE_PATH, chrome_instances_contents, overwrite=True)
```

Comment:
use write to json function

Reply:
this function is no longer in use, I'll just delete it